.github/workflows/e2e.yml

```diff
@@ -2,7 +2,7 @@ name: E2E
 
 on:
   pull_request:
-    branches: [ master, release/** ]
+    branches: [ master, release/**, eth-tx-interop ]
```

Comment:
Unsure, but I don't think you want to add this branch for e2e logic in the main branch github workflow right?